src/ml_board/frontend/newDashboard/src/app/App.tsx

```diff
@@ -0,0 +1,135 @@
+import { Component } from 'react';
```

Comment:
we still have the folder `newDashboard` in the paths.  
let's rename the folder to `dashbaord` and remove the old dashboard? @OsamaMSoliman what do you think?